floor_generator/test/processor/entity_processor_test.dart

```diff
@@ -482,6 +482,21 @@ void main() {
           throwsInvalidGenerationSourceError(
               EntityProcessorError(classElements).missingPrimaryKey));
     });
+    test('one primary key missing in compound primary key', () async {
```

Comment:
This sentence does not clearly say what is being tested here. I would say it like:
compound primary key element is not a column